src/table/body-cell/td-element.tsx

```diff
@@ -122,17 +122,19 @@ export const TableTdElement = React.forwardRef<HTMLTableCellElement, TableTdElem
         {...nativeAttributes}
         tabIndex={cellTabIndex === -1 ? undefined : cellTabIndex}
       >
-        {level !== undefined && isExpandable && (
-          <div className={styles['expandable-toggle-wrapper']}>
-            <ExpandToggleButton
-              isExpanded={isExpanded}
-              onExpandableItemToggle={onExpandableItemToggle}
-              expandButtonLabel={expandButtonLabel}
-              collapseButtonLabel={collapseButtonLabel}
-            />
-          </div>
-        )}
-        <span className={styles['body-cell-content']}>{children}</span>
+        <div className={styles['body-cell-content']}>
+          {level !== undefined && isExpandable && (
```

Comment:
It was necessary to move the expand toggle to inside the content to keep the changes to body-cell CSS minimal. In the future, we will need to move it back and adjust styles or maybe introduce another wrapper over `{children}`. That will be necessary to allow having both the expandable toggle and editable cell content in a single cell.